dask/dataframe/io/tests/test_parquet.py

```diff
@@ -15,6 +15,8 @@
 import dask.multiprocessing
 from dask.blockwise import Blockwise, optimize_blockwise
 from dask.dataframe._compat import PANDAS_GT_110, PANDAS_GT_121, PANDAS_GT_130
+from dask.dataframe.io.parquet.arrow import ArrowDatasetEngine
+from dask.dataframe.io.parquet.core import get_engine
```

Comment:
Since these imports require `pyarrow`, they're causing our minimum-dependency builds to fail. Since they're only used in one test, it probably makes sense to import them inside that test.